### PR TITLE
Release FX Devices v1.0beta9.3hotfix

### PR DESCRIPTION
**Pull request by BryanChi**
add desc line into FXD Saike BandSplitter

### CODE REVIEW

**Comment by cfillion** [comment]
1.0beta9.3hotfix is lower than 1.0beta9.3 so I've bumped it to 1.0beta9.3-1. Is removing the 'license: MIT' line intentional?

```lua
parse('1.0beta9.3')       --> { 1, 0, 'beta', 9, 3, 0s_until_infinity... }
parse('1.0beta9.3hotfix') --> { 1, 0, 'beta', 9, 3, 'hotfix', 0s_until_infinity... }
```

String segments such as 'hotfix' are sorted lower than 0 (1.0 > 1.0beta).

File: FX/BryanChi_FX Devices/FXD Saike BandSplitter.jsfx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 noindex: true
 
-license: MIT
+desc:FXD Saike BandSplitter
+noindex: true
```
Comment (cfillion) [inline FX/BryanChi_FX Devices/FXD Saike BandSplitter.jsfx]: This line is duplicated.
```suggestion
```

**Comment by BryanChi** [comment]
> 

I see, I'll keep that in mind for future requests. 
Sorry, removing 'license: MIT' was not intentional.

File: FX/BryanChi_FX Devices/FXD Saike BandSplitter.jsfx (hunk 1 above)
Comment (BryanChi) [reply FX/BryanChi_FX Devices/FXD Saike BandSplitter.jsfx]: my apologies, thanks for checking it for me!

**Pull request merged**